wallet/src/de/schildbach/wallet/ui/backup/RestoreWalletDialogFragment.java

```diff
@@ -274,6 +281,9 @@ public Dialog onCreateDialog(final Bundle savedInstanceState) {
             if (showEncryptedMessage) {
                 message.append("\n\n");
                 message.append(getString(R.string.restore_wallet_dialog_success_encrypted));
+            } else {
+                message.append("\n\n");
```

Comment:
Instead of adding a message here, we could rework how tutorial hints work and add something like a "security checklist" to the main flow.